tests/unit/test_pdf_reader.py

```diff
@@ -18,7 +19,7 @@ def pdf_reader(self):
         return PdfReader()
 
     def test_read_pdf(self, pdf_reader):
-        artifact = pdf_reader.get_content("./bitcoin.pdf")
+        artifact = pdf_reader.get_content(os.path.abspath("tests/unit/bitcoin.pdf"))
```

Comment:
@atcol should this maybe be `pdf_reader.get_content(os.path.abspath("./bitcoin.pdf"))`?